hive/replays/recurrent_replay.py

```diff
@@ -200,7 +200,7 @@ def sample(self, batch_size):
             trajectory_lengths = (
                 np.argmax(terminals.astype(bool), axis=1) + 1
             ) * is_terminal + self._n_step * (1 - is_terminal)
-            is_terminal = terminals[:,1:self._n_step-1]
```

Comment:
Because of the sequence length, is_terminal has to be of shape (B*seq_length). Hence this particular change after calculating trajectory lengths.